Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery.Tests/UnitTests.cs

```diff
@@ -0,0 +1,287 @@
+using NUnit.Framework;
+using System;
+using System.Threading;
+using System.Collections.Generic;
+using Frends.Oracle.ExecuteQuery.Definitions;
+using Newtonsoft.Json.Linq;
+
+namespace Frends.Oracle.ExecuteQuery.Tests
+{
+    /// <summary>
+    /// Docker container for Oracle database is needed to run these tests.
+    /// You need to have Docker installed before hand.
+    /// To create a docker container follow these instructions on Git CLI:
+    /// cd Frends.Oracle/Frends.Oracle.ExecuteQuery/build
+    /// ./deploy_oracle_docker_container.sh
+    /// Script might take a while (even 30 minutes) so relax.
+    /// To confirm that the container is up and running use Docker desktop or CLI. 
+    /// </summary>
+    [TestFixture]
+    [Ignore("Tests needs new Ubuntu based workflow")]
+    class TestClass
+    {
+        /// <summary>
+        /// Connection string for Oracle database.
+        /// </summary>
+        private static string _connectionString = "Data Source = (DESCRIPTION = (ADDRESS = (PROTOCOL = TCP)(HOST = localhost)(PORT = 51521))(CONNECT_DATA = (SERVICE_NAME = XEPDB1))); User Id = sys; Password=mysecurepassword; DBA PRIVILEGE=SYSDBA";
+        private static int _connectionTimeout = 30;
+
+        /// <summary>
+        /// Global variables.
+        /// </summary>
+        private static ConnectionProperties _connectionProperties;
+        private static QueryProperties _queryProperties;
+        private static Options _options;
+
+        [SetUp]
```

Comment:
Do you need to create the table before every test? I think table creation should only be done once, since there isn't a test which would drop the table.

Reply:
Changed the SetUp and TearDown to OneTimeSetUp and OneTimeTearDown